common/app/views/support/cleaner/AmpEmbedCleaner.scala

```diff
@@ -66,15 +70,26 @@ case class AmpEmbedCleaner(article: Article) extends HtmlCleaner {
       video.attr("layout", "responsive")
     }
 
+    def createFacebookVideoElement(document: Document, videoId: String, elementType: String): Element = {
+      val video = document.createElement(elementType)
+      video.attr("data-href", s"https://www.facebook.com/theguardian/videos/$videoId")
+      video.attr("data-embed-as", "video")
+      video.attr("width", "5")
+      video.attr("height", "3")
+      video.attr("layout", "responsive")
```

Comment:
Interesting idea! I will see what I can come up with on this one. Next commit is merging the create facebook element method into create element, but this seems like a more elegant way to do things. I will have a go at this tomorrow.